### ISSUE TITLE
Chat calls fail with TypeError: send_message() got an unexpected keyword argument 'context'

### ISSUE THREAD

**Issue by balajibr** [GoogleCloudPlatform/generative-ai#79]
Notebook: intro_langchain_palm_api.ipynb
LangChain version: 0.0.208
Vertex AI SDK version: 1.26.0

`chat([HumanMessage(content="Hello")]) ` 

fails with TypeError: send_message() got an unexpected keyword argument 'context'

Stack:
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[53], line 1
----> 1 chat([HumanMessage(content="Hello")])

File ~/.local/lib/python3.9/site-packages/langchain/chat_models/base.py:208, in BaseChatModel.__call__(self, messages, stop, callbacks, **kwargs)
    201 def __call__(
    202     self,
    203     messages: List[BaseMessage],
   (...)
    206     **kwargs: Any,
    207 ) -> BaseMessage:
--> 208     generation = self.generate(
    209         [messages], stop=stop, callbacks=callbacks, **kwargs
    210     ).generations[0][0]
    211     if isinstance(generation, ChatGeneration):
    212         return generation.message

File ~/.local/lib/python3.9/site-packages/langchain/chat_models/base.py:102, in BaseChatModel.generate(self, messages, stop, callbacks, tags, **kwargs)
    100 except (KeyboardInterrupt, Exception) as e:
    101     run_manager.on_llm_error(e)
--> 102     raise e
    103 llm_output = self._combine_llm_outputs([res.llm_output for res in results])
    104 generations = [res.generations for res in results]

File ~/.local/lib/python3.9/site-packages/langchain/chat_models/base.py:94, in BaseChatModel.generate(self, messages, stop, callbacks, tags, **kwargs)
     90 new_arg_supported = inspect.signature(self._generate).parameters.get(
     91     "run_manager"
     92 )
     93 try:
---> 94     results = [
     95         self._generate(m, stop=stop, run_manager=run_manager, **kwargs)
     96         if new_arg_supported
     97         else self._generate(m, stop=stop)
     98         for m in messages
     99     ]
    100 except (KeyboardInterrupt, Exception) as e:
    101     run_manager.on_llm_error(e)

File ~/.local/lib/python3.9/site-packages/langchain/chat_models/base.py:95, in <listcomp>(.0)
     90 new_arg_supported = inspect.signature(self._generate).parameters.get(
     91     "run_manager"
     92 )
     93 try:
     94     results = [
---> 95         self._generate(m, stop=stop, run_manager=run_manager, **kwargs)
     96         if new_arg_supported
     97         else self._generate(m, stop=stop)
     98         for m in messages
     99     ]
    100 except (KeyboardInterrupt, Exception) as e:
    101     run_manager.on_llm_error(e)

File ~/.local/lib/python3.9/site-packages/langchain/chat_models/vertexai.py:136, in ChatVertexAI._generate(self, messages, stop, run_manager, **kwargs)
    134 for pair in history.history:
    135     chat._history.append((pair.question.content, pair.answer.content))
--> 136 response = chat.send_message(question.content, **params)
    137 text = self._enforce_stop_words(response.text, stop)
    138 return ChatResult(generations=[ChatGeneration(message=AIMessage(content=text))])

TypeError: send_message() got an unexpected keyword argument 'context'

```

**Comment by edennuriel** [comment]
FWIW - it is working fine up to langchain 0.0.207

**Comment by pablofgaeta** [comment]
This issue has been fixed with the latest langchain version 0.0.212. 

Reference: https://github.com/hwchase17/langchain/pull/6652#issuecomment-1604938475

**Comment by balajibr** [comment]
Thanks, I confirm it works fine now.
